pkg/cmd/server/api/types.go

```diff
@@ -998,3 +1001,26 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string
 }
+
+// ClusterResourceOverrideConfig is the configuration for the ClusterResourceOverride
+// admission controller which overrides user-provided container request/limit values.
+type ClusterResourceOverrideConfig struct {
+	unversioned.TypeMeta
+	// Enabled must be true for the plugin to do anything.
+	// The plugin's actions can be disabled per-project with the project annotation
+	// quota.openshift.io/cluster-resource-override-enabled="false", so cluster admins
+	// can exempt infrastructure projects and such from the overrides.
+	Enabled bool
```

Comment:
That's not what I meant to ask :-) but I think I answered my own question. It looks like this plugin does nothing if the config is empty because all ratios are 0, making the Enabled flag superfluous.